foundations/installations/installations.md

```diff
@@ -260,10 +260,10 @@ If you encounter an error requesting you to disable **Intel RST** while attempti
 
 With the recent addition of being able to run a Linux terminal, the ChromeOS platform has been opened up to the ability to install native Linux applications. If you wish to use your Chromebook to complete The Odin Project, you will need to ensure you meet a couple requirements:
 
-1.  You have a supported Chromebook:
-    -   [Official Chromebooks](https://www.chromium.org/chromium-os/chrome-os-systems-supporting-linux)
-    -   [ChromeOS Flex Chromebooks](https://support.google.com/chromeosflex/answer/11513094)
-2.  You can install Linux by following [these instructions](https://support.google.com/chromebook/answer/9145439?hl=en)
+1. You have a supported Chromebook:
+    - [Official Chromebooks](https://www.chromium.org/chromium-os/chrome-os-systems-supporting-linux)
+    - [ChromeOS Flex Chromebooks](https://support.google.com/chromeosflex/answer/11513094)
```

Comment:
This should have a 3 space indent instead of 4

```suggestion
   - [Official Chromebooks](https://www.chromium.org/chromium-os/chrome-os-systems-supporting-linux)
   - [ChromeOS Flex Chromebooks](https://support.google.com/chromeosflex/answer/11513094)
```